### PR TITLE
BM Firmware Update

### PR DESCRIPTION
**Pull request by sreekarbvibm**

### CODE REVIEW

**Comment by ujjwal-ibm** [comment]
no documents change ?

File: ibm/service/vpc/resource_ibm_is_bare_metal_server.go [hunk 1]
```diff
@@ -217,6 +218,11 @@ func ResourceIBMIsBareMetalServer() *schema.Resource {
 				Computed:    true,
 				Description: "The CRN for this bare metal server",
 			},
+			isBareMetalServerFirmwareUpdateTypeAvailable: {
```
Comment (ujjwal-ibm) [inline ibm/service/vpc/resource_ibm_is_bare_metal_server.go:221]: missing documents change

Comment (sreekarbvibm) [reply ibm/service/vpc/resource_ibm_is_bare_metal_server.go:221]: Added

**Review by ujjwal-ibm — APPROVED** [review]
lgtm

**Pull request closed without merging**